e/etcd_bbolt/etcd_bbolt_ubi8.4.sh

```diff
@@ -0,0 +1,75 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : bbolt
+# Version       : v1.3.3
+# Source repo   : https://github.com/etcd-io/bbolt
+# Tested on     : RHEL ubi 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vikas Gupta <vikas.gupta8@ibm.com>
+#
```

Comment:
Since now onwards, please start adding a new filed for "Language" in header.
```
# Maintainer : Vikas Gupta <vikas.gupta8@ibm.com>
# Language : GO
```

Reply:
Added Language.